README.md

```diff
@@ -10,8 +10,11 @@ Before running you will have to set up an account at https://loggly.com and add
 
 Role Variables
 --------------
+Store the following in [a vault file](http://docs.ansible.com/ansible/playbooks_vault.html):
 
-LOGGLY TOKEN: The token associated with your loggly account
+    secrets:
```

Comment:
I'll try that - if [Galaxy](https://galaxy.ansible.com/amaabca/loggly_tls/) doesn't support it I might revert back to standard.